### DISCUSSION TITLE
2.0.2 - Fix King split on SURVEY

### DISCUSSION THREAD

**Discussion by Tenebrosful** [Tenebrosful/DELTARUNE-Livesplit.Autosplitter#43] [Announcements]
## What's Changed
* Fixed king end split not triggering on SP by @NERS1111 in https://github.com/Tenebrosful/DELTARUNE-Livesplit.Autosplitter/pull/41

**Full Changelog**: https://github.com/Tenebrosful/DELTARUNE-Livesplit.Autosplitter/compare/2.0.1...2.0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/Tenebrosful/DELTARUNE-Livesplit.Autosplitter/releases/tag/2.0.2'>2.0.2 - Fix King split on SURVEY</a>.</em>

**Comment by wille360creator** [comment]
hi i know this is a weird question and not about your post but its about deltarune i was wondering if you want to help me make a deltarune fan game.
